graphql_compiler/tests/test_data_tools/data_tool.py

```diff
@@ -27,6 +27,44 @@ def generate_orient_integration_data(client):
     _load_sql_files_to_orient_client(client, sql_files)
 
 
+def generate_neo4j_integration_data(client):
+    """Create Neo4j test DB from the SQL commands file for integration testing."""
+    project_root = path.dirname(path.abspath(__file__))
+    neo4j_files = glob(path.join(project_root, 'integration_data/create_neo4j_integration.cypher'))
+    _load_neo4j_files_to_neo4j_client(client, neo4j_files)
+
+
+def _load_neo4j_files_to_neo4j_client(client, neo4j_files):
```

Comment:
missing docstring?

Reply:
whoops, can't believe I missed that. Fixed now!